lib/ApiRequestor.php

```diff
@@ -254,9 +254,8 @@ private function _processResourceParam($resource, $hasCurlFile)
 
     private function _interpretResponse($rbody, $rcode, $rheaders)
     {
-        try {
-            $resp = json_decode($rbody, true);
-        } catch (Exception $e) {
+        $resp = json_decode($rbody, true);
+        if ($resp === null) {
```

Comment:
`json_last_error_msg()` is only available in PHP >= 5.5, but I updated the PR to use `json_last_error()`.